From b8f49dce799f0c177ee6a56b3fd67e7fdc6e68c2 Mon Sep 17 00:00:00 2001
Message-Id: <b8f49dce799f0c177ee6a56b3fd67e7fdc6e68c2.1650799331.git.geliang.tang@suse.com>
From: Wan Jiabing <wanjiabing@vivo.com>
Date: Tue, 12 Oct 2021 10:32:30 +0800
Subject: [PATCH] selftests/bpf: Remove duplicated include in cgroup_helpers
Git-commit: b8f49dce799f0c177ee6a56b3fd67e7fdc6e68c2
Patch-mainline: v5.16-rc1
References: bsn#127
Modified-by-SEL: No


Fix following checkincludes.pl warning:
./scripts/checkincludes.pl tools/testing/selftests/bpf/cgroup_helpers.c
tools/testing/selftests/bpf/cgroup_helpers.c: unistd.h is included more
than once.

Signed-off-by: Wan Jiabing <wanjiabing@vivo.com>
Signed-off-by: Andrii Nakryiko <andrii@kernel.org>
Acked-by: John Fastabend <john.fastabend@gmail.com>
Link: https://lore.kernel.org/bpf/20211012023231.19911-1-wanjiabing@vivo.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/bpf/cgroup_helpers.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/testing/selftests/bpf/cgroup_helpers.c b/tools/testing/selftests/bpf/cgroup_helpers.c
index 8fcd44841bb2..9d59c3990ca8 100644
--- a/tools/testing/selftests/bpf/cgroup_helpers.c
+++ b/tools/testing/selftests/bpf/cgroup_helpers.c
@@ -11,7 +11,6 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <ftw.h>
-#include <unistd.h>
 
 #include "cgroup_helpers.h"
 
-- 
2.26.2

